tests/ui/pages/discopane.py

```diff
@@ -0,0 +1,44 @@
+from pypom import Page
+from selenium.webdriver.common.by import By
+from selenium import webdriver
+import pytest
+
+
+class DiscoveryPane(Page):
+
+    _discopane_content_locator = (By.CSS_SELECTOR, '.disco-pane')
+    _play_video_locator = (By.CSS_SELECTOR, '.play-video')
+    _uninstalled_toggles = (By.CSS_SELECTOR, '.switch.uninstalled')
+    _close_video_link = (By.CSS_SELECTOR, '.close-video')
+    _see_more_addons_locator = (By.CSS_SELECTOR, '.amo-link')
+    _search_box_locator = (By.ID, 'lst-ib')
+
+    def open_addons(self):
+        elem = self.find_element(*self._search_box_locator)
+        elem.send_keys(Keys.COMMAND + Keys.SHIFT + 'a')
+
+    def play_video(self):
+        self.find_element(*self._play_video_locator).click()
+
+    def close_video(self):
+        self.find_element(*self._close_video_link).click()
+
+    @property
+    def is_video_playing(self):
+        return self.is_element_present(*self._close_video_link)
+
+    @property
+    def is_video_closed(self):
+        return self.is_element_present(*self._play_video_locator)
+
+    @property
+    def is_discopane_visible(self):
+        return self.is_element_present(*self._discopane_content_locator)
+
+    @property
+    def uninstalled_addons(self):
+        return self.find_elements(*self._uninstalled_toggles)
```

Comment:
Usually we wouldn't return `WebElements` to tests, but as we're just checking the length I think it's fine for now. I'd like to have an `AddOns` page region in a followup patch, which we can then return a list of.